ui-admin/src/study/StudySettings.tsx

```diff
@@ -18,7 +18,11 @@ export default function StudySettings({ studyEnvContext, portalContext }:
   const portalEnv = portalContext.portal.portalEnvironments
     .find(env =>
       env.environmentName === studyEnvContext.currentEnv.environmentName) as PortalEnvironment
-  return <div className="ps-4">
+
+  return <div className="container-fluid px-4 py-2">
+    <div className="d-flex mb-2">
+      <h2 className="fw-bold">Site Settings</h2>
+    </div>
```

Comment:
```suggestion
    { renderPageHeader('SiteSettings') } 
```
along with creating a new utility function
```
const renderPageHeader = (content: React.ReactNode) => {
  return <div className="d-flex mb-2">
      <h2 className="fw-bold">{content}</h2>
    </div>
}
```